modules/cart/src/Form/AddToCartForm.php

```diff
@@ -227,7 +264,7 @@ public function buildEntity(array $form, FormStateInterface $form_state) {
    * @return \Drupal\commerce_store\Entity\StoreInterface
    *   The selected store.
    */
-  protected function selectStore(PurchasableEntityInterface $entity) {
+  public function selectStore(PurchasableEntityInterface $entity) {
     $stores = $entity->getStores();
```

Comment:
@bojanz This works, but I wonder if we want to provide a custom form alter hook for add to cart forms, passing store and user explicitly. What do you think?